### ISSUE TITLE
Thoughts on a Live Lambda Development enhancement

### ISSUE THREAD

**Issue by rberger** [Plenituz/barbe-serverless#2]
Any thoughts on adding something like the [SST Live Lambda Development](https://docs.sst.dev/live-lambda-development) mechanism to Barbe-serverless?

**Comment by Plenituz** [comment]
Yeah this is definitely something we can, and plan to do. I think something similar can be done with `aws_fargate_task` as well. 
I'm bumping it up on my priority list
